.stylelintrc.yaml

```diff
@@ -30,7 +30,7 @@ rules:
   max-nesting-depth: 10
   no-descending-specificity: null
   # set to true to enable
-  plugin/stylelint-no-indistinguishable-colors: false
+  # plugin/stylelint-no-indistinguishable-colors: false
```

Comment:
Should these be commented? Or just removed?

Reply:
Just commented out. VSCode puts warning dialogues up about it requiring a certain version of stylelint, though, so I disable it locally to prevent those from popping up all the time.